PWGHF/HFC/Tasks/taskCorrelationDsHadrons.cxx

```diff
@@ -266,6 +267,16 @@ struct HfTaskCorrelationDsHadrons {
     }
   }
 
+  bool isSelectedCandidate(const int& ptBinD, const float& bdtScorePrompt, const float& bdtScoreBkg)
+  {
+
+    if (ptBinD == -1 || bdtScorePrompt < mlOutputPromptMin->at(ptBinD) || bdtScorePrompt > mlOutputPromptMax->at(ptBinD) || bdtScoreBkg > mlOutputBkg->at(ptBinD)) {
+      return false;
+    } else {
+      return true;
+    }
```

Comment:
If you are fine having all conditions in one line, it is more comprehensible to just return the value of the selection condition, i.e. the negation of what you have in `()`.